### ISSUE TITLE
Consider recommending FreeOTP+ fork

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-docs#403]
https://github.com/helloworld1/FreeOTPPlus appears to be a more actively maintained fork of FreeOTP with significant usability and functionality improvements. I suggest we consider recommending it over stock FreeOTP if there are no reasons not to.

**Comment by eloquence** [comment]
@huertanix Flagging for you visibility :)

**Comment by huertanix** [comment]
I'm not seeing an iOS version on helloworld1's repos unless I missed it somehow?

**Comment by eloquence** [comment]
The short version appears to be "not planned" (https://github.com/helloworld1/FreeOTPPlus/issues/215). That might be OK if we want to recommend it just to Android folk. The iOS version of FreeOTP at least still seems to be getting some attention (https://github.com/freeotp/freeotp-ios/releases), while the Android repo appears to have been inactive since May 2020.

**Comment by huertanix** [comment]
Yeah, I think there's the potential it can get _mildly_ confusing having two different recs dependent on platform. There's some cases for tooling where there's no choice but in this case, is there a must-have feature set that FreeOTPPlus offers that its predecessor does not? Or is there maybe a concern about apps that may not be as up to date on the security patching of their own software/library dependencies?

**Comment by eloquence** [comment]
I think there are two primary reasons to recommend the fork:
- https://github.com/freeotp/freeotp-android/issues/264 -- the official version right now displays a  warning upon start that it was built for an older Android version and may not function correctly. 
- FreeOTP+ has support for import/export of the database, which is really useful for account recovery, especially if you intentionally avoid including this data in cloud backups
